README.md

```diff
@@ -94,15 +98,36 @@ The interface is class with `__init__(settings: scrapy.Settings)` and `parse(fd:
 ```
 # settings.py
 SPIDERFEEDER_FILE_PARSERS = {
-    's3': 'myproject.my_custom_parser.CustomParser'
+    's3': 'myproject.my_custom_parser.parse'
```

Comment:
Why you're moving to functions here? I kinda liked the class-based parsers. You can add related code and auxiliar private methods to the class keeping the logic a little bit more organized and even inherit from other parsers.

Reply:
Just realized that a plain function with 2 parameters would be enough.
But I can go back in this change..

Reply:
It's up to you. We can always move back later if we need to.